pulse_actions/resultset_action.py

```diff
@@ -0,0 +1,7 @@
+from pulse_actions.worker import run_exchange_topic
```

Comment:
This file is to be called:
    python pulse_actions/resultset_action.py

for listening to pulse messages on exchange/treeherder/v1/resultset-actions exchange.